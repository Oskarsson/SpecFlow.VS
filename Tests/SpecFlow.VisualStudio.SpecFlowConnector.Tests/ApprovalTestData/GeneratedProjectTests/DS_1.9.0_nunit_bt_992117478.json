{
  "GeneratorOptions": {
    "FeatureFileCount": 1,
    "ScenarioPerFeatureFileCount": 1,
    "SpecFlowPackageVersion": "1.9.0"
  }
}
