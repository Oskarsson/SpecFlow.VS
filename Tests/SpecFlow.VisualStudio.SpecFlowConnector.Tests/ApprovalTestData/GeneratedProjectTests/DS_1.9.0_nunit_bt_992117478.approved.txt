{
  "stepDefinitions": [],
  "sourceFiles": {},
  "typeNames": {},
  "analyticsProperties": {
    "Connector": "<connector>",
    "ImageRuntimeVersion": "v4.0.30319",
    "SFFile": "TechTalk.SpecFlow.dll",
    "SFFileVersion": "1.9.0.77",
    "SFProductVersion": "1.9.0.77",
    "SourcePaths": "1",
    "StepDefinitions": "3",
    "TargetFramework": ".NETFramework,Version=v4.8",
    "TypeNames": "0"
  }
}